src/pharmpy/modeling/common.py

```diff
@@ -734,3 +734,17 @@ def get_model_attributes(model: Model, supress_warnings=False) -> str:
             if (e is not None and e != 0)
         ]
     )
+
+
+def filter_dataset(model, dv=1):
```

Comment:
The function needs to be exported in `modeling` and be given proper type hints and docstring.